test/fixtures/amd.after.js

```diff
@@ -0,0 +1,16 @@
+console.log('a');
```

Comment:
It would be sweet to have whitespace...I'm sure I can figure it out, becuase it *was* there before....